src/testdir/test_visual.vim

```diff
@@ -1523,4 +1523,15 @@ func Test_switch_buffer_ends_visual_mode()
   exe 'bwipe!' buf2
 endfunc
 
+func Test_heap_buffer_overflow()
```

Comment:
How about a comment which links to the bug?
Something like this?
```
" Check fix in Vim-9.0.????  for the heap-buffer-overflow bug reported at:
" https://huntr.dev/bounties/ae933869-a1ec-402a-bbea-d51764c6618e/
```
Otherwise it can be hard to know years later what was the goal of the test.

Reply:
agree, will fix